source/integrations-tools/external-resources.txt

```diff
@@ -0,0 +1,128 @@
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can find external tools and resources that you can use to
+enhance your {+odm+} applications. These resources include external projects and
+libraries, and various learning resources.
+
+Projects and Libraries
+----------------------
+
+The following sections describe projects and libraries that are integrated with
+or built on top of {+odm+}.
+
+Projects
+~~~~~~~~
+
+- `Workarea Commerce <https://github.com/workarea-commerce/workarea>`__ is an
+  enterprise-grade {+ror+} commerce platform that uses {+odm+}.
+
+Extension Libraries
+~~~~~~~~~~~~~~~~~~~
+  
+- `Mongoid Tree <https://github.com/benedikt/mongoid-tree>`__ is a tree structure
+  for {+odm+} documents that uses the materialized path pattern.
+  
+- `Mongoid Token <https://github.com/thetron/mongoid_token>`__ is a random,
+  unique token generator for {+odm+} documents.
+
+- `Mongoid Collection Snapshot
+  <https://github.com/mongoid/mongoid-collection-snapshot>`__ helps maintain
+  collections of processed data in {+odm+} applications.
+
+- `Mongoid Locker <https://github.com/mongoid/mongoid-locker>`__ provides
+  document-level locking for {+odm+} applications.
+  
+- `Mongo Beautiful Logger
+  <https://github.com/ibraheemdev/mongo_beautiful_logger>`__ is a library that
+  formats your MongoDB logs.
+  
+- `Mongoid Search <https://github.com/mongoid/mongoid_search>`__ provides
+  full-text search for {+odm+}.
+  
+- `Mongoid Fulltext Search <https://github.com/mongoid/mongoid_fulltext>`__
+  provides full-text search using n-gram matching for {+odm+}.
+
+Integration Libraries
+~~~~~~~~~~~~~~~~~~~~~
+
+- `CarrierWave Mongoid
+  <https://github.com/carrierwaveuploader/carrierwave-mongoid>`__ provides
+  {+odm+} support for the Carrierwave file-uploads library.
+
+- `Mongoid RSpec <https://github.com/mongoid/mongoid-rspec>`__ provide RSpec
+  matchers and macros for {+odm+} applications.
+
+- `RailsAdmin <https://github.com/sferik/rails_admin>`__ is a Rails engine that
+  provides an interface for managing your data.
+
+- `ActiveAdmin Mongoid <https://github.com/activeadmin/activeadmin-mongoid>`__
+  ActiveAdmin hacks to support {+odm+}.
+  
+- `Mongoid History <https://github.com/mongoid/mongoid-history>`__ is a
+  multi-user, non-linear history tracker for {+odm+}.
+
+- `Delayed Job Mongoid
+  <https://github.com/collectiveidea/delayed_job_mongoid>`__ is a {+odm+} backend for
+  ``delayed_job``.
+  
+- `Mongo Session Store <https://github.com/mongoid/mongo_session_store>`__ is a
+  Rails-compatible session store for {+odm+}.
+
+Learning Resources
+------------------
+
+The following sections provide screencasts, articles, and sample
+applications that you can use to learn more about {+odm+}.
+
+Screencasts
+~~~~~~~~~~~
+
+- `RailsCasts: Mongoid (revised)
+  <https://www.youtube.com/watch?v=L0RqU2MdqXU>`__ gives an overview of {+odm+},
+  by Ryan Bates. It includes the basics
```

Comment:
s: i would move 'by ryan bates' to somewhere else; part of the video link, or even a final sentence. maybe end the link with a colon, like in the Articles section, then 'An overview of {+odm+} by Ryan Bates.'